### PR TITLE
Fix account creation on email verification

### PR DESCRIPTION
**Pull request by altairrostam**
ATT-142
ATT-148

### CODE REVIEW

File: app/(protected)/dashboard/navItems.ts [hunk 1]
```diff
@@ -19,107 +19,107 @@ export type NavigationItems = {
...
-				icon: LogoutIcon,
-			},
-		],
-	},
+    {
+        group: "Patient Management",
+        children: [
+            {
+                text: "Patient Accounts",
+                link: "/dashboard/patient/accounts",
```
Comment (linerocks) [inline app/(protected)/dashboard/navItems.ts:27]: Please use enum for the links

File: app/(public)/accept-invitation/form/AcceptInvitationConfirmPhrase.tsx [hunk 2]
```diff
@@ -23,198 +23,198 @@ import { FormLabel, PhraseBox, PhraseChip } from "./styles";
...
+                    mt: 2,
+                    mb: 3,
+                }}
+            >
+                {shuffledPhrase?.map((item, i) =>
+                    includes(confirmPhrase, shuffledPhrase[i]) ? (
+                        <DisabledPhraseChip
+                            icon={
+                                <CheckIcon
+                                    sx={{ fontSize: 12, color: "#1C1B1F83" }}
```
Comment (linerocks) [inline app/(public)/accept-invitation/form/AcceptInvitationConfirmPhrase.tsx:171]: use color enum instead

File: app/(protected)/dashboard/navItems.ts (hunk 1 above)
Comment (altairrostam) [reply app/(protected)/dashboard/navItems.ts:27]: Will do another chore branch to fix this issue

File: app/(public)/accept-invitation/form/AcceptInvitationConfirmPhrase.tsx (hunk 2 above)
Comment (altairrostam) [reply app/(public)/accept-invitation/form/AcceptInvitationConfirmPhrase.tsx:171]: Will do another chore branch to fix this issue

**Pull request merged**